tests/NA_loss_funs.R

```diff
@@ -0,0 +1,124 @@
+NA_SegmentLoss <- function(NA_method){
```

Comment:
Why is this file in the `tests/` directory? This folder is reserved for unit test written with testthat. Please move all R code that the package should source into `R/`.